spec/spec_helper.rb

```diff
@@ -24,9 +24,9 @@
 
 ActiveRecord::ConnectionAdapters::SQLite3Adapter.class_eval do
   alias_method :exec_query_without_log, :exec_query
-  def exec_query(query, *args, &block)
+  def exec_query(query, *args, prepare: false, &block)
```

Comment:
would prefer if we can do `...` or `**kwargs` ... but this is good enough for now